orchagent/main.cpp

```diff
@@ -47,6 +47,7 @@ int gBatchSize = DEFAULT_BATCH_SIZE;
 bool gSairedisRecord = true;
 bool gSwssRecord = true;
 bool gLogRotate = false;
```

Comment:
>gLogRotate [](start = 5, length = 10)

The name seems cover gSaiRedisLogRotate. Do you mean `gOrchagentLogRotate` ?

Reply:
no, this logrotate only covers sairedis library